### DISCUSSION TITLE
Unable to run the code via Actions in Github, also I am not familiar with a Linux system

### DISCUSSION THREAD

**Discussion by saurabh4006** [shivamguys/irctc-cypress-automation#58] [Q&A]
![Error 1](https://github.com/user-attachments/assets/000f6db4-b5af-408f-ad34-ea2c71739cba)

I have never done coding and I am not at all familiar with Python or Java

I am trying to follow all the steps as suggested I have downloaded Virtual Box and I have installed Linux on VM, but when I try to run the code on Github by following the steps on the canva link I am getting stuck on this step 

Please help @shivamguys

**Comment by shivamguys** [comment]
Github Action is not working properly for this purpose have you tried
installing it on your virtual machine

**Reply by rajkmm** [reply]
Bro i am also getting the above error in my terminal 
I am using kali in vm
Please suggest...
Also have not used action and fork yet,  just git the repo and followed the steps of as you mentioned 
![image](https://github.com/user-attachments/assets/d9e99bc4-14eb-488d-b603-f1acbd6a5c84)
And here is the error...
![Screenshot_2024-07-23_08_19_28](https://github.com/user-attachments/assets/6e64f644-f524-443f-a019-bb8e7e0a17a1)

**Reply by saurabh4006** [reply]
![Screenshot from 2024-07-26 21-46-47](https://github.com/user-attachments/assets/77d29e0c-c146-4867-a1af-292a36a211ce)


Hi @shivamguys I have tried running it on VM Linux system but I am stuck on this step as shown in the above image as I am not sure what do you mean by # <---- Make Sure You Run This Command From Code Folder.  which particular folder is this ?? because I have tried running this command from all folders but I am still getting this error.

This might be a lame mistake but I have never done coding and I am stuck on this step and I desperately need to book a tatkal ticket at the earliest

Please respond at the earliest and if you could please mention it step by step then it would be great.

Thanks for your help!

**Reply by shivamguys** [reply]
Try to run commands only from irctc-cypress-automation folder, i do see you have moved into /cypress/e2e theres no need going over there...

Just install everything from that main folder only

**Reply by shivamguys** [reply]
> Bro i am also getting the above error in my terminal I am using kali in vm Please suggest... Also have not used action and fork yet, just git the repo and followed the steps of as you mentioned ![image](https://private-user-images.githubusercontent.com/54939703/351323797-d9e99bc4-14eb-488d-b603-f1acbd6a5c84.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIwMzM2NTEsIm5iZiI6MTcyMjAzMzM1MSwicGF0aCI6Ii81NDkzOTcwMy8zNTEzMjM3OTctZDllOTliYzQtMTRlYi00ODhkLWI2MDMtZjFhY2JkNmE1Yzg0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI2VDIyMzU1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlhNWI4N2ZjOWFmNTExMjNiMTQ4ZWEzZTRkOWU3YWMwNTFhOTE3MmEyN2MyZjZhNDExZDQwYWFmZWExYjk2ZWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.xjR-BDYuOUljLy2SO0B_CJ9oRVHhXXYK0tOR5C828Mo) And here is the error... ![Screenshot_2024-07-23_08_19_28](https://private-user-images.githubusercontent.com/54939703/351320854-6e64f644-f524-443f-a019-bb8e7e0a17a1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIwMzM2NTEsIm5iZiI6MTcyMjAzMzM1MSwicGF0aCI6Ii81NDkzOTcwMy8zNTEzMjA4NTQtNmU2NGY2NDQtZjUyNC00NDNmLWEwMTktYmI4ZTdlMGExN2ExLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI2VDIyMzU1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ1OGEyZDJhODhkYzUxMDJlNTczYzhlMDAzMmI2NzM1ODVkZWJiNTFjMDNhMjAzZjAwZTUxMjllNGZlOWUyMDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.i6x6FtmSuedi7_ryJgiAnRahv0Ev1oR86cR8_IL3RoE)

Did it worked?

**Comment by shivamguys** [comment]
https://github.com/shivamguys/irctc-cypress-automation/discussions/1#discussioncomment-10161828


@saurabh4006 please have a look
